shell/platform/darwin/ios/framework/Headers/FlutterEngine.h

```diff
@@ -133,6 +133,34 @@ FLUTTER_EXPORT
  */
 - (void)destroyContext;
 
+/**
+ * Ensures that Flutter will generate a semantics tree.
+ *
+ * This is enabled by default if certain accessibility services are turned on by
+ * the user, or when using a Simulator. This method allows a user to turn
+ * semantics on when they would not ordinarily be generated and the performance
+ * overhead is not a concern, e.g. for UI testing. Note that semantics should
+ * never be programatically turned off, as it would potentially disable
+ * accessibility services an end user has requested.
+ *
+ * This method must only be called after launching the engine via
+ * `-runWithEntrypoint:` or `-runWithEntryPoint:libraryURI`.
+ */
+- (void)ensureSemanticsEnabled;
```

Comment:
Nomenclature question: Is there a reason that this is more idiomatic than `enableSemantics`?

Reply:
I'm trying to make it clear that this is not something you could ever try to set to false.  It's a one way, turn on only method.  It would be dangerous to let people arbitrarily turn off semantics.

Reply:
Right, I didn't mean `enableSemantics(bool)` but just `enableSemantics()` which would also imply one-way.